gkeys/gkeys/actions.py

```diff
@@ -15,6 +15,10 @@
 
 import os
 import sys
+import time
+import re
+import smtplib
+from email.mime.text import MIMEText
 
```

Comment:
Hmm, if these new imports are only needed for the emails, then perhaps they should be put in the demandload.